### PR TITLE
VoiceFlow User story

### PR DESCRIPTION
**Pull request by Anita-ihuman**

### CODE REVIEW

**Comment by eyalb181** [comment]
User photo and logo aren't showing here: https://e03c8a71.mirrord-dev.pages.dev/stories/

File: content/stories/voiceflow/index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "VoiceFlow User Story | Greg O'Grady"
```
Comment (eyalb181) [inline content/stories/voiceflow/index.md:2]: You're missing a profile pic and an avatar. Look at the other user stories to see exactly how and where to put them. Note the profile pic has to be a cropped circle of the pic you added - you can probably use some online tool for it like this thing: https://crop-circle.imageonline.co/

Comment (eyalb181) [inline content/stories/voiceflow/index.md:2]:
```suggestion
title: "Voiceflow User Story | Greg O'Grady"
```

File: content/stories/voiceflow/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+card_title: "Greg O'Grady"
+description: "mirrord abstracts away the complexities for developers who may not have deep knowledge of Kubernetes or the networking stack"
+date: 2024-11-7T06:00:00+00:00
+lastmod: 2024-11-7T06:00:00+00:00
+position: "Software Engineer | VoiceFlow"
```
Comment (eyalb181) [inline content/stories/voiceflow/index.md:7]: Fix all instances of VoiceFlow to Voiceflow

**Review by eyalb181 — CHANGES_REQUESTED** [review]

**Comment by Anita-ihuman** [comment]
@eyalb181 I believe I have gone through these reviews. Anymore comments?

**Comment by Anita-ihuman** [comment]
> Please see if you can find a logo with the name of the company. Also it should be smaller - have a look at what it looks like in the preview site.

updated.

**Review by eyalb181 — APPROVED** [review]
Approved, well done

**Pull request merged**